cmd/registry/cmd/annotate/annotate.go

```diff
@@ -47,7 +47,7 @@ func Command() *cobra.Command {
 			}
 			args[0] = c.FQName(args[0])
 
-			client, err := connection.NewRegistryClient(ctx)
+			client, err := connection.NewRegistryClientWithSettings(ctx, c)
```

Comment:
Just curious: since `c` is just `connection.ActiveConfig()` (essentially available to anything with access to the `connection` package), why did this function signature need to change? (couldn't `connection.NewRegistryClient()` just use the active config as its default?)

Reply:
Yes, it can. The issue is that we're loading it twice. I could, instead, cache it in a global once loaded and simply retrieve it from that. What do you think?

Reply:
That sounds good, but optional -- I don't think we need to hold this PR for that.

Reply:
Ok, thanks.